### PR TITLE
Added CSS for the Get Concordance, Languages, Generate Tokens

### PR DESCRIPTION
**Pull request by HimanshuBCS**
1. Added CSS for the Get Concordance, Languages, Generate Tokens.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -43,8 +50,8 @@ input.active:invalid {
  }
 
   .source-header-lan {
```
Comment (udkumar) [inline src/App.css]: make proper alignment

File: src/GetLanguages.js [hunk 2]
```diff
@@ -101,7 +101,7 @@ class GetLanguages extends Component {
             <div className="container">
             <div className="floatLeft">
               <table className="table">
-                <thead>
+                <thead >
```
Comment (udkumar) [inline src/GetLanguages.js]: Why you put space here?

File: src/App.css (hunk 1 above)
Comment (HimanshuBCS) [inline src/App.css]: Done.

File: src/GetLanguages.js (hunk 2 above)
Comment (HimanshuBCS) [inline src/GetLanguages.js]: Done.

**Pull request merged**